app/models/service_template/copy.rb

```diff
@@ -5,7 +5,7 @@ def template_copy(new_name = "Copy of " + name + Time.zone.now.to_s)
     if template_valid? && type != 'ServiceTemplateAnsiblePlaybook'
       ActiveRecord::Base.transaction do
         dup.tap do |template|
-          template.update_attributes(:name => new_name, :display => false)
+          template.update!(:name => new_name, :display => false, :options => {:button_order => []})
```

Comment:
@d-m-u How does the button_order get updated to cb-#{id}? Is there a rails callback that's updating it?

Reply:
that's what the rest of the lines in this file are doing